website/content/docs/connect/config-entries/ingress-gateway.mdx

```diff
@@ -1399,9 +1424,11 @@ The following table describes the configurations for passive health checks:
 
 | Parameter | Description | Data type | Default |
 | ---       | ---         | ---       | ---     |
-| `interval` |  Specifies the time in nanoseconds between checks. | Integer | Proxy's default configuration, which is `10` for Envoy |
-| `max_failures` | Specifies the number of consecutive failures that cause a host to be removed from the upstream cluster. | Integer | Proxy's default configuration, which is `5` for Envoy |
-| `enforcing_consecutive_5xx` | Specifies a percentage that indicates how many times out of 100 that Consul ejects the host when it detects an outlier status. The outlier status is determined by consecutive errors in the 500-599 response range. | Integer | Proxy's default configuration, which is `100` for Envoy |
+  | `Interval` | Specifies the time between checks. | string | `0s` |
+  | `MaxFailures` | Specifies the number of consecutive failures allowed per check interval. If exceeded, Consul removes the host from the load balancer. | integer | `0` |
+  | `EnforcingConsecutive5xx` | Specifies a percentage that indicates how many times out of 100 that Consul ejects the host when it detects an outlier status. The outlier status is determined by consecutive errors in the 500-599 response range. | integer | `100` |
+  | `MaxEjectionPercent` | The maximum % of an upstream cluster that can be ejected due to outlier detection. Defaults to 10% but will eject at least one host regardless of the value. | integer | `10` |
+  | `BaseEjectionTime` | The base time that a host is ejected for. The real time is equal to the base time multiplied by the number of times the host has been ejected and is capped by max_ejection_time (Default 300s). Defaults to 30000ms or 30s. | string | `30s` |
```

Comment:
Just to flag these as sections that need to have consistent language.

Reply:
Done